packages/core/src/Site/index.ts

```diff
@@ -887,15 +959,18 @@ class Site {
   /**
    * Handles the rebuilding of modified pages
    */
-  async handlePageReload(oldAddressablePages, oldPagesSrc, oldSiteConfig) {
+  async handlePageReload(oldAddressablePages: AddressablePage[], oldPagesSrc: string[],
+                         oldSiteConfig: SiteConfig) {
     this.collectAddressablePages();
 
     // Comparator for the _differenceWith comparison below
-    const isNewPage = (newPage, oldPage) => _.isEqual(newPage, oldPage) || newPage.src === oldPage.src;
+    const isNewPage = (newPage: AddressablePage,
+                       oldPage: AddressablePage) => _.isEqual(newPage, oldPage)
```

Comment:
Minor nit but I would prefer if the RHS expression of `=>` is all on the same line.